spec/factories/card.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :card do
+    original_text "How are you?"
+    translated_text "Kamusta?"
+    review_date DateTime.now.getutc
```

Comment:
@davetoxa DateTime.now returns local time but 3.days.from_now returns UTC time. once i convert these using .to_date i get rounding errors in my tests where my cards will be due 4 days (because i'm in GMT+8)